packages/cashier/src/pages/on-ramp/on-ramp-provider-card/__tests__/on-ramp-provider-card.spec.tsx

```diff
@@ -1,34 +1,41 @@
 import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import OnRampProviderCard from '../on-ramp-provider-card';
-
-jest.mock('Stores/connect', () => ({
-    __esModule: true,
-    default: 'mockedDefaultExport',
-    connect: () => Component => Component,
-}));
+import { StoreProvider } from '@deriv/stores';
+import { TRootStore } from 'Types';
 
 describe('<OnRampProviderCard />', () => {
-    const props = {
-        is_dark_mode_on: false,
-        is_mobile: false,
-        provider: {
-            name: 'Changelly',
-            icon: {
-                dark: 'IcCashierChangellyDark',
-                light: 'IcCashierChangellyLight',
-            },
-            getDescription: jest.fn(
-                () =>
-                    'Your simple access to crypto. Fast and secure way to exchange and purchase cryptocurrencies. 24/7 live chat support.'
-            ),
-            getPaymentIcons: jest.fn(() => [{ dark: 'IcCashierFpsDark', light: 'IcCashierFpsLight' }]),
+    const provider = {
+        name: 'Changelly',
+        icon: {
+            dark: 'IcCashierChangellyDark',
+            light: 'IcCashierChangellyLight',
         },
-        setSelectedProvider: jest.fn(),
+        getDescription: jest.fn(
+            () =>
+                'Your simple access to crypto. Fast and secure way to exchange and purchase cryptocurrencies. 24/7 live chat support.'
+        ),
+        getPaymentIcons: jest.fn(() => [{ dark: 'IcCashierFpsDark', light: 'IcCashierFpsLight' }]),
     };
 
     it('should show proper messages and button', () => {
-        render(<OnRampProviderCard {...props} />);
+        const mockRootStore: DeepPartial<TRootStore> = {
```

Comment:
@farzin-deriv could we refactor `mockRootStore` to avoid duplication? For example create `mockRootStore` in `beforeEach` function and in test cases just change the required field. (e.g `is_dark_mode_on` in 'should show proper icons in dark_mode' test case)?